src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2103 @@
+| E4000 | Relationship: `{0}` cannot link to itself | |
+| E4001 | Relationship Item `{0}` for Relationship `{1}` is invalid: an Item can link only one Tracker entity. | |
+| E4006 | Could not find relationship Type: `{0}`. | |
+| E4009 | Relationship Type `{0}` is not valid. | |
+| E4010 | Relationship Type `{0}` constraint requires a {1} but a {2} was found. | |
+| E4011 | Relationship: `{0}` cannot be persisted because {1} {2} referenced by this relationship is not valid. | |
+| E4012 | Could not find `{0}`: `{1}`, linked to Relationship. | |
+| E4013 | Relationship Type `{0}` constraint is missing {1}. | |
+| E4014 | Relationship Type `{0}` constraint requires a Tracked Entity having type `{1}` but `{2}` was found. | |
+| E9999 | N/A | Undefined error message. |
+
+### Validation
+
+<!--DHIS2-SECTION-ID:webapi_nti_validation-->
+
+While importing data using the tracker importer, a series of validations are performed to ensure the validity of the data. This section will describe some of the different types of validation performed to provide a better understanding if validation fails for your import.
+
+#### Required properties
+Each of the tracker objects has a few required properties that need to be present when importing data. For an exhaustive list of required properties, have a look at the [Tracker Object section](#webapi_nti_tracker_objects).
+
+When validating required properties, we are usually talking about references to other data or metadata. In these cases, there are three main criteria:
+1. The reference is present and not null in the payload.
+2. The reference points to the correct type of data and exists in the database
+3. The user has access to see the reference
+
+If the first condition fails, the import will fail with a message about a missing reference. However, suppose the reference points to something that doesn't exist or which the user cannot access. In that case, both cases will result in a message about the reference not being found.
+
+#### Formats
+Some of the properties of tracker objects require a specific format. When importing data, each of these properties is validated against the expected format and will return different errors depending on which property has a wrong format. Some examples of properties that are validated this way:
+- UIDs (These cover all references to other data or metadata in DHIS2.)
+- Dates
+- Geometry (The coordinates must match the format as specified by its type)
+
+#### User access
+All data imported will be validated based on the metadata  (Sharing) and the organisation units (Organisation Unit Scopes) referenced in the data. You can find more information about sharing and organisation unit scopes in the following sections.
+
+Sharing is validated at the same time as references are looked up in the database. Metadata outside of the user's access will be treated as if it doesn't exist. The import will validate any metadata referenced in the data.
```

Comment:
Mention data sharing level as well? Or at least link to it?

Reply:
Added links